scout/server/blueprints/cases/controllers.py

```diff
@@ -418,58 +417,6 @@ def case_report_content(store, institute_id, case_name):
     return data
 
 
-def coverage_report_contents(store, institute_obj, case_obj, base_url):
```

Comment:
This is the code required to capture the content of the coverage report. This content is at the moment attached to the PDF general report , so you export both reports at the same time. This is a bit dangerous when you have for instance general report oriented ad a portrait and coverage report as landscape. Then the second gets cut and doesn't fit the page.

By removing the coverage report from the PDF general one it will force users to manually download both, but I think this is an OK compromise!

Reply:
Yes, all info is there. We just need to communicate it so there is no confusion on the users part. They will have to adjust their dl pattern slightly. Chanjo(-report) still needs an overhaul, so the ability to smoothly integrate with another report could go on the list of requirements there. Just being able to query / extract a few key values to include would be neato.